packages/agreement-process/src/services/agreementService.ts

```diff
@@ -173,6 +175,26 @@ export function agreementServiceBuilder(
         await repository.createEvent(event);
       }
 
+      return streamId;
+    },
+    async cloneAgreement(
+      agreementId: string,
+      authData: AuthData
+    ): Promise<string> {
+      logger.info("Cloning agreement");
```

Comment:
Same as this comment here: https://github.com/pagopa/interop-be-monorepo/pull/136#discussion_r1439500454